pyproject.toml

```diff
@@ -27,12 +27,16 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = "^3.7"
 getostheme = "<2023,>=2021"
-Pillow = "<10,>=9.1.0"
-PySimpleGUI = "<5,>=4.59.0"
+#Pillow = "<10,>=9.1.0"
+Pillow = ">=9"
```

Comment:
Please try and use project tooling to manage this, setting Pillow = ">=9" breaks semver (even if not a perfect system)